### DISCUSSION TITLE
About SNAREseq data analysis

### DISCUSSION THREAD

**Discussion by YinCH97** [stuart-lab/signac#807] [Q&A]
Hi,
    When I download the SNARE instance tutorial data for scATAC analysis, but has been unable to use the fragment file.
    When I run TSSEnrichment,I get this error[Error in base::rowSums(x, na.rm = na.rm, dims = dims, ...) :   'x' must be an array of at least two dimensions ]
    When I run GeneActivity(snare), I get this error:Error in SingleFeatureMatrix(fragment = fragments[[x]], features = features,  :  No matching chromosomes found in fragment file.
    Is there something wrong with the fragment file?

**Comment by YinCH97** [comment]
` chrom_assay@fragments
[[1]]
A Fragment object for 10309 cells`

**Comment by timoast** [comment]
Can you show the full code you're running and output of `sessionInfo()`?
